.github/workflows/tests.yml

```
@@ -0,0 +1,100 @@
+name: Run CI tests
```

```suggestion
name: Tests
```

Just a nit here. I think the "run/ci" bits are redundant?